__device-tests__/gutenberg-editor-sanity-test-1-visual.test.js

```diff
@@ -263,7 +263,8 @@ describe( 'Gutenberg Editor - Test Suite 1', () => {
 			);
 			const cellSize = await cell.getSize();
 			const cellLocation = await cell.getLocation();
-			const scrollOffset = isAndroid() ? 400 : 150;
+			const scrollOffset = isAndroid() ? 350 : 100;
+			const windowSize = await editorPage.driver.getWindowSize();
```

Comment:
Hey @dcalhoun 👋 I ended up changing this to scroll to half of the window size height value instead of the `scrollOffset` value, I kept getting failures on CI due to scroll differences of a 1px value.

Do you have any concerns about this change? Let me know. Thanks!

Reply:
No concerns. This test is a bit finicky. Scroll distance and inertia can cause subtly different outcomes when scrolling past the end of the container view. Updating the distance makes sense to me.

Reply:
Awesome! thanks you!